adagucserverEC/CImgRenderPolylines.cpp

```diff
@@ -143,29 +193,56 @@ void CImgRenderPolylines::render(CImageWarper *imageWarper, CDataSource *dataSou
               cnt++;
             } else {
               CDBDebug("status: %d %d [%f,%f]", status, j, tprojectedX, tprojectedY);
```

Comment:
Should we not remove all these Debug statements?

Reply:
Done